### ISSUE TITLE
Page menu / navigation buttons' focus outline cut off

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#7813]
## Bug Description
There isn't enough space currently for displaying the focus outline for the buttons below the Page / Page navigation buttons.

## Expected Behaviour
Focus outline should be fully visible

## Steps to Reproduce
Tab into Page buttons / navigation

## Screenshots
![Screenshot 2021-06-03 at 10 16 27](https://user-images.githubusercontent.com/3294597/120603600-80a83d00-c44c-11eb-87dd-7dc03c14829f.png)
![Screenshot 2021-06-03 at 10 16 10](https://user-images.githubusercontent.com/3294597/120603613-83a32d80-c44c-11eb-8900-975eef7c77ba.png)

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Verified in QA
